unstructured/ingest/main.py

```diff
@@ -392,6 +395,12 @@ def run(self):
     show_default=True,
     help="Number of parallel processes to process docs in.",
 )
+@click.option(
+    "--recursive",
```

Comment:
we have local-recursive and drive-recursive flags currently, so it might be misleading to have a non-specific recursive flag that impacts specifically some of the fsspec connectors. that said, wondering, should we elevate recursive to a StandardConnectorConfig field?

Reply:
Yes. I've been thinking about this. I will look into this.

Reply:
Looks like we can just use one flag
--recursive
for fsspec, local, and google drive

Doesn't seem like it belongs in StandardConnectorConfig until we can confirm that it works for all connectors, since StandardConnectorConfig seems to be more related to the second step after connecting (like where to download, etc...)

Would be nice to see if we can get it working on all connectors, but that seems out of scope for right now. Though some connectors would never need a recursive (like wikipedia...)

Thoughts?

Reply:
Yea, I don't know that all of the StandardConnectorConfig options always pertain to _all_ of the connectors (for instance none of the download fields impact the local connector). That said, probably `remote_url` is a closer parallel for what we're trying to do here (overlap a collection of connectors) and it's more as you suggest (generic parameter that gets mapped to each connector-specific config). So totally good with that, as long as we only end up with one flag --recursive.

Reply:
One flag to rule them all.